users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -1320,6 +1331,160 @@ def run_specaug_stft_experiments():
     return report, ctc_nn_system
 
 
+def run_scf_combination_experiments():
+    gs.ALIAS_AND_OUTPUT_SUBDIR = "experiments/switchboard/ctc/feat/"
+
+    _, nn_system_stft_specaug = run_specaug_stft_experiments()
+
+    (
+        returnn_datasets,
+        rasr_loss_corpus_path,
+        rasr_loss_corpus_segments,
+        rasr_loss_lexicon_path,
+        dev_corpora,
+    ) = get_datasets(use_multi_proc_dataset=True, pre_process=CodeWrapper("audio_perturb_runner.run"))
+    returnn_args = {
+        "batch_size": 5000,
+        "rasr_binary_path": RASR_BINARY_PATH,
+        "rasr_loss_corpus_path": rasr_loss_corpus_path,
+        "rasr_loss_corpus_segments": rasr_loss_corpus_segments,
+        "rasr_loss_lexicon_path": rasr_loss_lexicon_path,
+        "datasets": returnn_datasets,
+        "conformer_type": "wei",
+        "audio_perturbation": True,
+        "use_multi_proc_dataset": True,
+        "extra_args": {
+            "accum_grad_multiple_step": 2,
+            "watch_memory": True,
+            "conv_pad_seq_len_to_power": 1.5,
+            "audio_perturb_runner": CodeWrapper("WaveformPerturbation(**audio_perturb_args)"),
+        },
+    }
+
+    # usually the lr args would need to be adapted to fit with the checkpoint,
+    # but experiments showed that restarting the scheduling is better
+    lr_args = {
+        "peak_lr": 4e-4,
+        "start_lr": 1.325e-05,
+        "end_lr": 1e-5,
+        "increase_epochs": 180,
+        "decrease_epochs": 180,
+        "final_epochs": 0,
+    }
+
+    feature_args_scf = {"class": "ScfNetwork", "size_tf": 256 // 2, "stride_tf": 10 // 2, "preemphasis": 0.97}
+
+    def _preload_dict_helper(train_name, epoch):
+        return {
+            "existing-model": {
+                "filename": nn_system_stft_specaug.train_jobs[train_name].out_checkpoints[epoch],
+                "init_for_train": True,
+            },
+        }
+
+    nn_base_args = {
+        "scf_bs2x5k_stft20ms_fmask_5_8of512_tempo": dict(
+            returnn_args={
+                **returnn_args,
+                "specaug_stft": {
+                    "max_feature": 8,
+                    "frame_size": 400,
+                    "frame_shift": 160,
+                    "fft_size": 512,
+                },
+                "extra_args": {
+                    **returnn_args["extra_args"],
+                    "audio_perturb_args": {"tempo": {"prob": 1, "minimum": 0.7, "maximum": 1.3}},
+                    "preload_from_files": {
+                        **_preload_dict_helper("conformer_bs2x5k_scf_stft20ms_fmask_5_8of512", 24)
+                    },
```

Comment:
Wouldn't this be the same and more concise?
```suggestion
                    "preload_from_files": _preload_dict_helper("conformer_bs2x5k_scf_stft20ms_fmask_5_8of512", 24),
```